sage-ios/SAGE/SAGE/EditAnnouncementController.swift

```diff
@@ -52,4 +58,22 @@ class EditAnnouncementController: AddAnnouncementController {
         }
     }
 
+    func deleteAnnouncement(sender: UIButton!) {
+        let view = (self.view as! AddAnnouncementView)
```

Comment:
wait just kidding....but the editView should be a property of the controller so you can just set self.view = self.editView in loadView